react/week2/todolist/src/todo-app.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from "react";
+import "./App.css";
+import todos from "./todos";
+function Todo({ item, reMove, styling }) {
+  return (
+    <li>
+      <h3 className={item.done === false ? "" : "checkedText"}>
+        {item.description}
+        <input type="checkbox" onChange={styling}></input>
+      </h3>
+      <button onClick={reMove}>Delete</button>
+    </li>
+  );
+}
```

Comment:
You can extract this to a separate file. In this small application it's fine of course, but when you start working on your final project, you'll see that your files can grow very big if you don't separate components. 
One of the big advantages of React is that you can reuse components in many part of your application (again this doesn't in this case). That you can only achieve, if you have components in separate files. 
So it's just something to keep in mind for the future.

Reply:
you are absolutely right , we are now facing that in the final project and its a very good hint to remember. thanks again!